test-engine-core-modules/requirements.txt

```diff
@@ -13,7 +13,7 @@ h5py==3.9.0 ; python_version >= "3.10" and python_version < "3.12"
 idna==3.4 ; python_version >= "3.10" and python_version < "3.12"
 jax==0.4.13 ; python_version >= "3.10" and python_version < "3.12"
 joblib==1.2.0 ; python_version >= "3.10" and python_version < "3.12"
-keras==2.12.0 ; python_version >= "3.10" and python_version < "3.12"
```

Comment:
I'll probably need to drop bumping of Tensorflow/Keras from this PR, or create a new one with this specific change. 

The change is in here to make tests run on my MacBook.